docs/docs/dcap.md

```diff
@@ -19,19 +9,33 @@ automated for you.
 As cloud providers run caches, DCAP supports vendor-specific plugins. Intel provides a default one
 which requires a [subscription](https://api.portal.trustedservices.intel.com/provisioning-certification).
 
-Azure provides a DCAP [plugin](https://github.com/microsoft/Azure-DCAP-Client) that does not require a subscription. Conclave
-bundles and uses Azure's DCAP plugin by default. The Azure caches are open to the public internet and can be used from anywhere. Azure Confidential Computing instances like DC4s_v3 come pre-provisioned for DCAP and as Conclave
+Azure provides a DCAP [plugin](https://github.com/microsoft/Azure-DCAP-Client) that does not require a subscription. 
+Conclave bundles and uses Azure's DCAP plugin by default. The Azure caches are open to the public internet and can 
+be used from anywhere. Azure Confidential Computing instances like DC4s_v3 come pre-provisioned for DCAP and as Conclave
 comes with the necessary libraries bundled, you don't need to do any further setup.
 
+DCAP is also known as ECDSA attestation.
+
 ### DCAP Plugin
-To perform attestation using DCAP, Conclave needs a way to gather information about the platform the enclave is hosted on. This information provides proof from Intel that the system supports SGX and that it is patched and up to date.
+To perform attestation using DCAP, Conclave needs a way to gather information about the platform the enclave is 
+hosted on. This information provides proof from Intel that the system supports SGX and that it is patched and up to 
+date.
+
+DCAP is designed to work on different server topologies. So, rather than directly connecting to Intel services to 
+retrieve this information, the cloud vendor or the owner of the SGX system must provide a DCAP client plugin with 
```

Comment:
> So, rather than directly connecting to Intel services to 
retrieve this information

This sentence is implying the EPID protocol, and so I think we should bring that to attention here and use it to mention that we don't support it. So something like

"So, rather than directly connecting to Intel services to retrieve this information, like in the old EPID protocol which is no longer supported, the cloud vendor ..."